examples/step-20/step-20.cc

```diff
@@ -253,8 +253,8 @@ namespace Step20
   void KInverse<dim>::value_list(const std::vector<Point<dim>> &points,
                                  std::vector<Tensor<2, dim>> &  values) const
   {
-    Assert(points.size() == values.size(),
-           ExcDimensionMismatch(points.size(), values.size()));
+    (void) points;
+    AssertDimension(points.size(), values.size());
```

Comment:
Do we use the `(void)var` idiom in previous examples? Do we need to explain it?

Reply:
We use it in `step-9` and `step-14` and explain it in the latter one. It certainly doesn't hurt to explain the idiom here as well. I updated the PR accordingly.